astropy/units/format/base.py

```diff
@@ -125,9 +119,21 @@ def _format_inline_fraction(
             return f"{scale}/ {denominator}"
         return f"{scale}{numerator} / {denominator}"
 
+    @classmethod
+    def _format_multiline_fraction(
+        cls, scale: str, numerator: str, denominator: str
+    ) -> str:
+        # By default, we just warn that we do not have a multiline format.
+        warnings.warn(
+            f"format {cls.__name__} does not support multiline "
+            "fractions; using inline instead.",
+            UnitsWarning,
+        )
+        return cls._format_inline_fraction(scale, numerator, denominator)
+
     @classmethod
     def to_string(
-        cls, unit: UnitBase, *, fraction: bool | Literal["inline"] = True
+        cls, unit: UnitBase, *, fraction: bool | Literal["inline", "multiline"] = True
```

Comment:
Now that all the formatters accept the same values for `fraction`, we could define a [type alias](https://docs.python.org/3.11/library/typing.html#type-aliases) (but we don't have to).